settings/src/main/scala/org/typelevel/sbt/TypelevelSettingsPlugin.scala

```diff
@@ -201,7 +205,29 @@ object TypelevelSettingsPlugin extends AutoPlugin {
         Seq("-Werror")
       else
         Seq.empty
+    },
+    scalacOptions ++= {
+      val releaseOption = if (isJava8) Seq() else Seq("-release", tlJvmRelease.value.toString)
+      val targetOption = if (isJava8) Seq() else Seq(s"-target:${tlJvmRelease.value}")
```

Comment:
On 2.12.15 it seems the formatting is different?
```
  -target:<target>                     Target platform for object files. All JVM 1.5 - 1.7 targets are deprecated. (jvm-1.5,jvm-1.6,jvm-1.7,[jvm-1.8])
```

Reply:
Done.